blog/forms.py

```diff
@@ -0,0 +1,17 @@
+"""Creating and editing forms."""
+from django import forms
+
+from .models import Post
+
+
+class PostForm(forms.ModelForm):
+    """Class form based on the model."""
+
+    class Meta:
+        """Choose a model for creation form.
```

Comment:
Does linter really require docstrings for in-body classes?

Also, it's not an action, but metadata specification/description.